packages/react-components/source/react/library/tag/Tag.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import { func, string, oneOf, bool } from 'prop-types';
+import { func, string, oneOf, bool, node } from 'prop-types';
 import classNames from 'classnames';
 import Button from '../button';
 import Text from '../text';
 
 const propTypes = {
   /** Tag text or other content */
-  label: string.isRequired,
+  label: node.isRequired,
```

Comment:
Shouldn't this be `label: PropTypes.oneOfType([node, string]);` to preserve existing behavior? Or does node cover this case somehow?

Reply:
PropTypes.node includes "anything that can be rendered: numbers, strings, elements or an array" so this should be fine.

Reply:
awesome! DIdn't know that. Thanks